src/rail/estimation/algos/deepdisc.py

```diff
@@ -156,7 +188,7 @@ def run(self):
         output_dir = self.config.output_dir
         output_name = self.config.output_name
 
-        val_per = 5
+        val_per = 5 # Should be included as an input config parameter.
```

Comment:
Change this to an input parameter, check with @grantmerz to see what it should be called.

Reply:
Eval_period is fine

Reply:
Tracked in https://github.com/lincc-frameworks/deepdisc/issues/75

Reply:
Referenced in issue #23